### PR TITLE
Prettified the client page a bit, added links and the ManageIQ gettin…

### PR DESCRIPTION
**Pull request by theute**
…g started blog as a doc

### CODE REVIEW

**Comment by theute** [comment]
Don't merge yet, we'll merge #194 first after review and hopefully this time I won't screw the rebase

**Comment by theute** [comment]
Please someone review and merge

File: src/main/jbake/content/hawkular-clients/index.adoc [hunk 1]
```diff
@@ -1,48 +1,96 @@
...
+|
+|Metrics API
+|
+
+|*Ruby*
+|link:https://github.com/hawkular/hawkular-client-ruby/blob/master/README.rdoc[README]
+|link:https://github.com/hawkular/hawkular-client-ruby[GitHub repository]
+|
+|Metrics API,
+Inventory API
```
Comment (pilhuhn) [inline src/main/jbake/content/hawkular-clients/index.adoc:77]: Also Alerts/Events and Operations.
I pretend this is the most advanced library and under very active development by the Hawkular team itself.

Comment (theute) [inline src/main/jbake/content/hawkular-clients/index.adoc:77]: "most advanced library and under very active development by the Hawkular team itself." --> Correct.
I may move Android Client and HawkFx to that page (From community/labs) but we should express some form of readyness/activity statement...
Now the labs would only contain datamining which is more like a server extension...

**Pull request merged**